example.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+// helloWorld is a function that does stuff 
```

Comment:
```suggestion
// helloWorld {{ funcComment }}
```

Change `funcComment` by commenting `@codelingobot set funcComment <your comment>`

Reply:
@CodeLingoBot set funcComment "some comment"

Reply:
Thanks, I updated `{{ funcComment }}` to "some comment" run `@codelingobot set funcComment "your comment"` again to update.

Reply:
@CodeLingoBot are you active?

Reply:
@CodeLingoBot hello?

Reply:
@CodeLingoBot now?

Reply:
@CodeLingoBot set funcComment "simply prints out "Hello, World!" it's kind of a thing."

Reply:
@CodeLingoBot set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"

Reply:
@CodeLingoBot rewrite set funcComment "is super basic a conventional"